builder/oracle/oci/config.go

```diff
@@ -101,7 +101,10 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 
 	if c.Region != "" {
 		accessCfg.Region = c.Region
-	} else {
+	}
+
+	// Default if the template nor the API config contains a region.
+	if accessCfg.Region == "" {
 		accessCfg.Region = "us-phoenix-1"
```

Comment:
As an aside, we could move these into constants to prevent using strings everywhere.

Reply:
I think we should do a follow up PR to remove the region defaulting (to match the change to the OCI terraform provider).